pytorch_blade/pytorch_blade/torch-mlir/lib/Dialect/TorchConversion/Transforms/ApplyValueSemantics.cpp

```diff
@@ -89,6 +90,16 @@ class ApplyValueSemanticsPass
       op->erase();
     }
 
+    // fix NonValueTensor type operator
+    func.walk([&](Operation* op) {
+      if (isa<OverwriteTensorContentsOp>(op)) {
+        rewriter.setInsertionPoint(op);
+        auto nonValueTensor = rewriter.create<CopyToNonValueTensorOp>(
```

Comment:
Why do we need to convert `OverwriteTensorContentsOp` to `CopyToNonValueTensorOp` and then to `disc.ArgsMutation`? Is it possible to skip `CopyToNonValueTensorOp`?

Reply:
Maybe `CopyToNonValueTensorOp ` is required, the function `reduceTensorConversions(func)` at L79 removs all Tensor conversations and convert all `!tensor` to `!vtensor`,  but `OverwriteTensorContentsOp ` operand requires `!tensor` , so we need to insert `CopyToNonValueTensorOp` to make `OverwriteTensorContentsOp ` validation after `reduceTensorConversions`.